### DISCUSSION TITLE
List of css classes

### DISCUSSION THREAD

**Discussion by bernhardh** [filamentphp/filament#386] [Help]
I am new to the tall stack and tailwind, so maybe its a stupid question: Can I find somewhere all available css classes used in filament (except of the minified css file)?

As far as I see, not all classes described on https://tailwindcss.com/ are available. For example I couldn't use `mr-2` or `mr-3` or `ml-1` or `ml-2`, etc. but `mr-1` or `ml-3` is available. That seems not really consistend to me.

**Comment by ryangjchandler** [comment]
> I am new to the tall stack and tailwind, so maybe its a stupid question: Can I find somewhere all available css classes used in filament (except of the minified css file)?
> 
> As far as I see, not all classes described on https://tailwindcss.com/ are available. For example I couldn't use `mr-2` or `mr-3` or `ml-1` or `ml-2`, etc. but `mr-1` or `ml-3` is available. That seems not really consistend to me.

The reason some classes are available and others aren't is because the Tailwind is being purged to reduce the size of the file. We would recommend importing your own CSS if Filament doesn't already have the class available.

**Reply by bernhardh** [reply]
Thanks. I understand the reason behind that, but is there somehow a list to see what is available out of the box? I would prefere to stick to the classes already available, if possible. Currently I am `ctrl+f` in the minified version of the css file.

**Reply by ryangjchandler** [reply]
@bernhardh There's not a list no. That list would be huge - for now, you're better off with trial and error or as you said, <kbd>Ctrl + F</kbd> in the minified version.
